Rakefile

```
@@ -38,7 +38,7 @@ task default: :test
 begin
   require 'rubocop/rake_task'
   RuboCop::RakeTask.new
-rescue
+rescue StandardError
```

You can be more specific
```suggestion
rescue LoadError
```